sleep-for-days.md

```diff
@@ -0,0 +1,81 @@
+### Go
+---
+```
+// Send an email every 30 days
+func SleepForDaysWorkflow(ctx workflow.Context) (string, error) {
+  for {
+    workflow.ExecuteActivity(ctx, SendEmail) // Activities have timeouts, and will be retried by default!
+    workflow.Sleep(ctx, time.Hour*24*30) // Sleep for 30 days!
+  }
+  // ...
```

Comment:
```suggestion
```

Now that we are using infinite loops, there's nothing that can happen after it, so this "// ..." makes less sense (same for other langs)

Reply:
Per the other comment about activity configs, might add this _before_ the loop instead for all langs (some langs would *require* this for activity options, but if we do it for some I might do it for all for consistency)

Reply:
Yeah, signalling there may be code before the loop makes sense to me